add, get and delete work

Leohemmingsson opened the pull request:
# Description
Add, get, and delete files work. So the basic functionality with SQL and ABE is fixed.

# What is not fixed?
The program will sometimes return the  *wrong* status code. This is due to not checking for all errors.
The program can not change between different schemes either.


fixes ivario123/NGAC_ABE/#18
fixes ivario123/NGAC_ABE/#20
fixes ivario123/NGAC_ABE/#35

Klas-Norling commented:
The load_static_keys_from_sql function in aw11 and kpac17 needs to convert the static keys to PyAw11GlobalKey respectively PyAc17PublicKey and PyAc17MasterKey.

Remove unused code in src/sending_request.py (if your not going to use the code that has been commented out)

Even though you the function names in setup_sql_tables.py is pretty self explanitory you might want to add comments.

Leohemmingsson commented:
@Klas-Norling I have fixed comments in `setup_sql_tables.py`.
Further, I have refactored the code in `sending_request.py` so it is divided into proper functions and looks much less bulky.
But I will fix `KpAc17` and `AW11` in another PR. This is to make it possible to merge a functioning version a little faster.
